### PR TITLE
[CR] Delete docs migrated to wiki

### PR DESCRIPTION
**Pull request by illi-kun**
Currently we have some info for developers in wiki and other part of it in `\doc` folder.

I've moved two md files to project's wiki (no changes, just copy-paste).

These pages now available here:
https://github.com/CleverRaven/Cataclysm-DDA/wiki/Developer-FAQ
https://github.com/CleverRaven/Cataclysm-DDA/wiki/Game-Balance

I did it because now it's possible to copy this wiki to your PC for working with it offline (so you get the same MD files as previously) but updating of it will be easier (at least I guess so, correct me if I wrong).

I can easily do the same for the rest of files if this change moves us to correct direction.

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
How are updates proposed now?

**Comment by illi-kun** [comment]
Are write rights restricted to contributors only?

**Comment by BorkBorkGoesTheCode** [comment]
It looks like it.

**Comment by mugling** [comment]
I don't think it gets much use under `doc/`. Very few new contributors actually read it there whereas a carefully constructed wiki would be much more useful

**Comment by BorkBorkGoesTheCode** [comment]
>a carefully constructed wiki would be much more useful

+1

Is write access for the wiki separate from the project?

**Comment by mugling** [comment]
No idea, but the documentation is updated so infrequently it's not a significant concern here. Having it online on the wiki might encourage developers to quickly document their work as they go.

**Comment by illi-kun** [comment]
> Is write access for the wiki separate from the project?

Yes, and it can be changed as described [here](https://help.github.com/articles/changing-access-permissions-for-wikis/).

File: doc/GAME_BALANCE.md [hunk 1]
```diff
@@ -1,254 +0,0 @@
...
-
-- glass bottle: 3
-
-- plastic canteen: 6
-
-- 3L glass jar: 12, as expected
-
-- gallon jug: 15
-
-# Diamond weapons
```
Comment (mugling) [inline doc/GAME_BALANCE.md]: Tangent: this is now probably outdated

**Comment by kevingranade** [comment]
This was a mistake, edits to the wiki are not reviewed, which is critical for balance guidelines, and the wiki is basically invisible.

**Pull request merged**